### PR TITLE
feat: add view cashier component

### PR DESCRIPTION
**Pull request by BLasan**
## Description
View Cashier component created.

## Related issues and discussion
#761 

## Screenshots, if any
Screenshot from 2020-05-22 14-16-28

## Checklist
Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] If you have multiple commits please combine them into one commit by squashing them.

- [x] Read and understood the contribution guidelines at `web-app/.github/CONTRIBUTING.md`.

### CODE REVIEW

File: src/app/organization/tellers/view-cashier/view-cashier.component.html [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+</div>
+
+
+<div class="container">
+
+  <mat-card>
+
+    <mat-card-content>
+
+      <h2 class="mat-h2" fxFlexFill>{{ cashierData.staffName}}</h2>
```
Comment (karantakalkar) [inline src/app/organization/tellers/view-cashier/view-cashier.component.html:15]: Add name field as in view holiday

File: src/app/organization/tellers/view-cashier/view-cashier.component.html [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+        <div fxFlex="50%">
+          {{ cashierData.tellerName }}
+        </div>
+
+        <div fxFlex="50%" class="header">
+          Description
+        </div>
+
+        <div fxFlex="50%">
```
Comment (karantakalkar) [inline src/app/organization/tellers/view-cashier/view-cashier.component.html:33]: make it conditional if data not present

File: src/app/organization/tellers/view-cashier/view-cashier.component.ts [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+   * Get cashier data from `Resolver`.
+   * @param {ActivatedRoute} route Activated Route.
+   */
+  constructor(private route: ActivatedRoute) {
+    this.route.data.subscribe((data: { cashier: any }) => {
+      this.cashierData = data.cashier;
+    });
+  }
+
+  ngOnInit() {
```
Comment (karantakalkar) [inline src/app/organization/tellers/view-cashier/view-cashier.component.ts:28]: remove unused code

File: src/app/organization/tellers/view-cashier/view-cashier.component.html (hunk 2 above)
Comment (BLasan) [reply src/app/organization/tellers/view-cashier/view-cashier.component.html:33]: You mean if the value is empty, the Description and its value both should be removed?

**Comment by BLasan** [comment]
@karantakalkar Please review

File: src/app/organization/tellers/view-cashier/view-cashier.component.html [hunk 4]
```diff
@@ -0,0 +1,71 @@
+<div fxLayout="row" fxLayoutAlign="end" fxLayoutGap="2%" fxLayout.lt-md="column" class="container m-b-20">
+  <button mat-raised-button color="primary" [routerLink]="['edit']">
```
Comment (karantakalkar) [inline src/app/organization/tellers/view-cashier/view-cashier.component.html:2]: dont add routerlink until component is ready

File: src/app/organization/tellers/view-cashier/view-cashier.component.html [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+        <div fxFlex="50%" *ngIf="cashierData.description !== ''">
+          {{ cashierData.description }}
+        </div>
+
+        <div fxFlex="50%" class="header">
+          Start Date
+        </div>
+
+        <div fxFlex="50%">
+          {{ cashierData.startDate | date:'yyyy-MM-dd' }}
```
Comment (karantakalkar) [inline src/app/organization/tellers/view-cashier/view-cashier.component.html:46]:
```suggestion
          {{ cashierData.startDate | date}}
```

File: src/app/organization/tellers/view-cashier/view-cashier.component.html [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+        </div>
+
+        <div fxFlex="50%" class="header">
+          End Date
+        </div>
+
+        <div fxFlex="50%">
+          {{ cashierData.endDate | date:'yyyy-MM-dd' }}
```
Comment (karantakalkar) [inline src/app/organization/tellers/view-cashier/view-cashier.component.html:54]:
```suggestion
          {{ cashierData.endDate | date }}
```

File: src/app/organization/tellers/view-cashier/view-cashier.component.scss [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+}
+
+h2, h3, h4 {
+  margin: 0;
+  font-weight: 500;
+}
+
+mat-divider {
+  margin: 0 0 0.5em 0;
+}
```
Comment (karantakalkar) [inline src/app/organization/tellers/view-cashier/view-cashier.component.scss:23]: remove unused scss make it similar to view employee

File: src/app/organization/organization-routing.module.ts [hunk 8]
```diff
@@ -56,6 +56,8 @@ import { EditOfficeComponent } from './offices/edit-office/edit-office.component
 import { AdhocQueryTemplateResolver } from './adhoc-query/adhoc-query-template.resolver';
 import { ViewLoanProvisioningCriteriaComponent } from './loan-provisioning-criteria/view-loan-provisioning-criteria/view-loan-provisioning-criteria.component';
 import { LoanProvisioningCriteriasResolver } from './loan-provisioning-criteria/loan-provisioning-criterias.resolver';
+import { ViewCashierComponent } from './tellers/view-cashier/view-cashier.component';
```
Comment (karantakalkar) [inline src/app/organization/organization-routing.module.ts:59]: why has component been placed with resolvers?

File: src/app/organization/tellers/view-cashier/view-cashier.component.html (hunk 4 above)
Comment (BLasan) [reply src/app/organization/tellers/view-cashier/view-cashier.component.html:2]: ok

File: src/app/organization/organization-routing.module.ts (hunk 8 above)
Comment (BLasan) [reply src/app/organization/organization-routing.module.ts:59]: Automatically imported. I'll place it in correct posiotion

**Comment by BLasan** [comment]
@karantakalkar Done

**Pull request merged**